static/js/chart/draw.ts

```diff
@@ -904,7 +1011,8 @@ function drawGroupBarChart(
   chartHeight: number,
   dataGroups: DataGroup[],
   formatNumberFn: (value: number, unit?: string) => string,
-  options?: ChartOptions
+  options?: ChartOptions,
+  useLollipop?: boolean
```

Comment:
can you embed this to the above options object?

Reply:
Good idea. Done!